src/TypeScript/Analyzer/JavaScriptAnalyzer.cs

```diff
@@ -155,25 +150,16 @@ internal async Task ExecuteAnalysis(string filePath, IIssueConsumer consumer, Ca
             catch (Exception ex) when (!ErrorHandler.IsCriticalException(ex))
             {
                 analysisStatusNotifier.AnalysisFailed(filePath, ex);
+                RequireLinterUpdate();
             }
         }
 
         private async Task EnsureEslintBridgeClientIsInitialized(CancellationToken cancellationToken)
         {
-            var port = await eslintBridgeProcess.Start();
-
             try
             {
                 serverInitLocker.WaitOne();
 
-                if (port != javascriptServerPort)
-                {
-                    javascriptServerPort = port;
-                    shouldInitLinter = true;
```

Comment:
Why isn't `shouldInitLinter` `true` when the analyzer is created? It knows it needs to call it, and that it hasn't already called it.